src/data/char.lean

```diff
@@ -8,7 +8,7 @@ Authors: Mario Carneiro
 # More `char` instances
 
 > THIS FILE IS SYNCHRONIZED WITH MATHLIB4.
-> https://github.com/leanprover-community/mathlib4/pull/609
+> https://github.com/leanprover-community/mathlib4/pull/None
```

Comment:
I'm not sure why the yaml file no longer includes 
* https://github.com/leanprover-community/mathlib4/pull/609